Assignment1 - Code Review Changes

megha994 opened the pull request:
Changes made as per code review comments

ericnjanga commented on _pages/8.html:
```diff
@@ -0,0 +1,398 @@
+                      page 10 in the 2023-2026 Strategtic Business Plan</span
+                    ></a
+                  >
+                </p>
```
Please move this paragraph outside of the list. The designer made an alignment mistake. In fact, the paragraph should be on the same vertical line as the list. Please do the same for the other 3 boxes' paragraphs.

megha994 replied:
fixed

ericnjanga commented on _pages/8.html:
```diff
@@ -0,0 +1,398 @@
+        <!--------------->
+        <div class="col-md-12 col-sm-12 bx-container">
+          <div class="bottom-section-spacer-sm">
+            <h2 id="01" class="bottom-section-spacer-sm">
```
Please remove the class "bottom-section-spacer-sm" here, The h2's natural bottom spacing is enough.

Note: Once again, it's not your fault, the designer made a little exaggeration.

megha994 replied:
fixed

ericnjanga commented on _pages/8.html:
```diff
@@ -0,0 +1,398 @@
+
+          <div class="bottom-section-spacer-sm">
+            <h2 id="01" class="bottom-section-spacer-sm">
+              <span class="modified-heading">
```
Please remove the class "bottom-section-spacer-sm" here, The h2's natural bottom spacing is enough.

Note: Once again, it's not your fault, the designer made a little exaggeration.

megha994 replied:
fixed

ericnjanga commented on _pages/8.html:
```diff
@@ -0,0 +1,398 @@
+        <div class="col-md-7 col-sm-7 bx-container">
+          <div id="1" class="bottom-section-spacer-sm">
+            <h2>
+              <span class="modified-heading"
```
I'm not sure why you created this class. Once, I understand more about your thoughts, I'll let you know.

megha994 replied:
oke

ericnjanga reviewed (changes requested):
Please address my comments!

megha994 committed:
Merge pull request #3 from megha994/main

Merge pull request #2 from megha994/assignment1

megha994 committed:
Merge branch 'assignment1' of https://github.com/megha994/Case-Study-3 into assignment1

megha994 committed:
modified page 8 according to requested changes

megha994 committed:
Merge branch 'main' of https://github.com/megha994/Case-Study-3 into assignment1

megha994 committed:
Merge remote-tracking branch 'upstream/main' into assignment1

megha994 committed:
1. Color of all links changed
2. Navigation enabled for all links
3. Guidelines droupdown content added
4. All contents on page synced with original page
